pytorch_lightning/metrics/functional/classification.py

```diff
@@ -73,16 +55,9 @@ def stat_scores_multiple_classes(
     reduction: str = 'none',
 ) -> Tuple[torch.Tensor, torch.Tensor, torch.Tensor, torch.Tensor, torch.Tensor]:
     """
-    Calculates the number of true positive, false positive, true negative
-    and false negative for each class
-
-    .. warning :: Deprecated in favor of :func:`~pytorch_lightning.metrics.functional.stat_scores`
-
-    Raises:
-        ValueError:
-            If ``reduction`` is not one of ``"none"``, ``"sum"`` or ``"elementwise_mean"``.
+    .. deprecated::
+        Use :func:`torchmetrics.functional.stat_scores`. Will be removed in v1.4.0.
```

Comment:
```suggestion
        Use :func:`torchmetrics.functional.stat_scores_multiple_classes`. Will be removed in v1.4.0.
```

Reply:
this is not a valid func in `torchmetrics`